src/patternfly/components/Banner/banner.scss

```
@@ -10,6 +10,8 @@
   --#{$banner}--FontSize: var(--pf-t--global--font--size--body--default);
   --#{$banner}--Color: var(--pf-t--global--text--color--nonstatus--on-gray--default);
   --#{$banner}--BackgroundColor: var(--pf-t--global--color--nonstatus--gray--default);
+  --#{$banner}--BorderColor: var(--pf-t--global--border--color--default);
```

Did you mean to use the HC border color here?

```suggestion
  --#{$banner}--BorderColor: var(--pf-t--global--border--color--high-contrast);
```